tests/ui/use_last.rs

```diff
@@ -0,0 +1,36 @@
+#![warn(clippy::use_last)]
+
+fn dont_use_last() -> Option<i32> {
+    let x = vec![2, 3, 5];
+    let last_element = x.get(x.len() - 1); // ~ERROR Use x.last()
+    last_element.map(|val| val + 1) // To avoid warnings
+}
+
+fn indexing_two_from_end() -> Option<i32> {
+    let x = vec![2, 3, 5];
+    let last_element = x.get(x.len() - 2);
+    last_element.map(|val| val + 3) // To avoid warnings
+}
+
+fn index_into_last() -> i32 {
+    let x = vec![2, 3, 5];
+    let last_element = x[x.len() - 1];
+    last_element + 1 // To avoid warnings
+}
+
+// False positive test (currently failing)
```

Comment:
This is probably not relevant anymore?

Also: https://github.com/rust-lang/rust-clippy/pull/3832/files#r262524555